aspnetcore/security/docker-https.md

```diff
@@ -109,9 +109,11 @@ Run the container image with ASP.NET Core configured for HTTPS:
 
 ```console
 docker pull mcr.microsoft.com/dotnet/core/samples:aspnetapp
-docker run --rm -it -p 8000:80 -p 8001:443 -e ASPNETCORE_URLS="https://+;http://+" -e ASPNETCORE_HTTPS_PORT=8001 -e ASPNETCORE_Kestrel__Certificates__Default__Password="password" -e ASPNETCORE_Kestrel__Certificates__Default__Path=\https\aspnetapp.pfx -v %USERPROFILE%\.aspnet\https:C:\https\ mcr.microsoft.com/dotnet/core/samples:aspnetapp
+docker run --rm -it -p 8000:80 -p 8001:443 -e ASPNETCORE_URLS="https://+;http://+" -e ASPNETCORE_HTTPS_PORT=8001 -e ASPNETCORE_Kestrel__Certificates__Default__Password="$CREDENTIAL_PLACEHOLDER$" -e ASPNETCORE_Kestrel__Certificates__Default__Path=c:\https\aspnetapp.pfx -v %USERPROFILE%\.aspnet\https:C:\https\ mcr.microsoft.com/dotnet/core/samples:aspnetapp
 ```
 
+***NOTE:*** `$CREDENTIAL_PLACEHOLDER$`is a placeholder for the Kestrel certificates default password.
```

Comment:
This line uses `$CREDENTIAL_PLACEHOLDER$` as the placeholder. But up on line 55, it uses `<password>`. And on line 122, it uses `{ password here }`. This all needs to get cleaned up to be consistent. The same syntax and placeholder name should be used in all cases. I prefer the angle bracket syntax for placeholders.

There are two other usages of `ASPNETCORE_Kestrel__Certificates__Default__Password="password"` that need to be updated as well.